### DISCUSSION TITLE
Reverse Relations

### DISCUSSION THREAD

**Discussion by Dazeh** [pocketbase/pocketbase#1724] [Q&A] [answered]
Hey,

If i have posts, which has a relation of user. When i query said user, how can i get posts of that user? 

`pb.collection('posts').getFullList(200, {expand: 'user'})`

The above gets my all posts with the user expanded. But if i do the following: 

`pb.collection('user').getList(1,1, { expand: 'posts'})`

I get users, but not expanded posts. 

Thanks for your help!

**Comment by ganigeorgiev** [comment, marked as answer]
For indirect expand you can check https://pocketbase.io/docs/expanding-relations/#indirect-expand.

In your case, the above `user` posts expand should be something like: `expand: 'posts(user)'`.

Alternatively, if you want to fetch only the user's posts, you can also use a `filter`:

```js
pb.collection('posts').getFullList(200, {filter: `user = '${userId}'`})
```
